service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -702,6 +702,7 @@ public boolean hasFutureHearing(List<Element<HearingBooking>> hearingBookings) {
     }
 
     private final DocumentReference submittedForm;
+    private final DocumentReference draftApplicationDocument;
```

Comment:
interesting. was it never referred?

Reply:
no, never, it is just put to caseDetails map, but never read back